docs/technical-overview.md

```diff
@@ -0,0 +1,267 @@
+
+We use fixture features for the following purposes:
+- Generate a possibly short variety of test fixtures that use possibly many different fixture features so we can check all special cases
+- *(planned)* The fixture editor can only edit/import fixtures that only use editor-compatible fixture features
+- *(planned)* Search for fixtures with specific fixture features (mainly for testing)
+
+Fixture features are saved in the the `cli/fixture-features/` directory as JS modules that export an array of features. It is adviced to put similar features into one module. A sample module looks like this:
+
+```js
+module.exports = [{
+  // Optional. Used internally and in test-fixtures.json.
+  // Default is the filename (without '.js'), succeded by `-${i}` if multiple features per module are provided.
+  id: 'fine-channel-alias',
+
+  // Required. Try to be as short as possible as it's used in generated table header. Markdown is allowed.
+  name: 'Fine channels (16bit)',
+
+  // Optional. Is used as tooltip in generated table header. Markdown is not allowed.
+  description: 'Whether a channel defines exactly one fine channel alias',
+
+  // Optional. The feature with the highest order is in the first (most-left) column.
+  // Default value is 0, negative values are allowed (to appear right to the features with default order).
+  order: 81,
+
+  /**
+   * Required. Checks if the given fixture uses this module's feature.
+   * @param {!Fixture} fixture The Fixture instance, see our model.
+   * @return {!Boolean} true if fixture uses the feature
+   */
+  hasFeature: fixture =>
+    fixture.availableChannels.some(
+      channel => channel.fineChannelAliases.length === 1
+    )
+}];
+```
+
+## Plugins
+*[⬆️ Back to top](#technical-overview)*
+
+The aim of OFL is to import and export our fixture definitions from/to fixture formats of third-party lighting controlment software, for example [QLC+](https://github.com/mcallegari/qlcplus)'s `.qfx` format. A plugin is a converter between our and an external format. It implements an import and/or export method that processes/generates the third-party format.
+
+Each plugin has its own directory `plugins/<plugin-key>/` which contains all information, methods and tests about the external format. Please provide a `README.md` as markdown file with a short explanation about the fixture format. If applicable, it should include
+
+* a link to the software that uses this format
+* how to import fixtures into the software
+* a place where fixtures of this format can be obtained from
+
+### Exporting
+*[⬆️ Back to top](#technical-overview)*
+
+If exporting is supported, create a `plugins/<plugin-key>/export.js` module that exports the plugin name, version and a method that generates the needed third-party files out of an given array of fixtures. This method should return an array of objects for each file; the files are zipped together automatically. A file object looks like this:
```

Comment:
* "that exports" is confusing here
* "This method should return an array of objects for each file that should be exported/downloadable; the files are zipped together automatically if neccessary."